src/test/test.py

```diff
@@ -124,14 +125,24 @@ def write_config_files(self, output_dir):
 class TestArchive(TestBase):
 
     def test_archive(self):
-        out_dir = self.tmp_dir
         json_path = os.path.join(self.sup_dir, 'report_json', 'WGTS', 'djerba_report.json')
-        archive_path = archiver().run(json_path, out_dir, 'test_ID')
-        self.assertTrue(os.path.exists(archive_path))
-        # contents of file are dependent on local paths
-        with open(archive_path) as archive_file:
-            data = json.loads(archive_file.read())
-        self.assertEqual(len(data['report']), 21)
+        archive_status = archiver().run(json_path)
```

Comment:
We will want to configure a separate database for unit tests and/or use a mock database object, instead of using the production DB -- but this is OK for now